Makefile

```diff
@@ -32,6 +34,11 @@ clean:
 install:
 	poetry install --extras "docs"
 
+develop:
+	pip install poetry
```

Comment:
I don't think this line should be included, as there are [many ways to install](https://python-poetry.org/docs/) poetry.

Reply:
ideally, `Makefile` should make project ready to run. So far, installing `poetry` via pip looks a single cross platform solution.

Reply:
The Poetry developers [don't recommend](https://python-poetry.org/docs/#alternative-installation-methods-not-recommended) installing using pip. However, the contributor's guide could be updated to include a link to the Poetry installation page.

Reply:
Let me update the docs then